stripe/_http_client.py

```diff
@@ -568,6 +595,11 @@ def close(self):
         pass
 
 
+class _Proxy(TypedDict):
```

Comment:
There is another copy of this type in the file https://github.com/stripe/stripe-python/blob/8f1168417aa0815e1e9f6229dca031a9ae72d34c/stripe/_http_client.py#L107